scripts/minici-presubmit-all-tests.sh

```diff
@@ -28,6 +29,7 @@ case $1 in
         export SETUP_SCRIPT="scripts/minikube-minishift-setup-env.sh"
         export RUN_SCRIPT="scripts/minikube-minishift-all-tests.sh"
         export TIMEOUT="4h00m"
+        export CI_INFO="minikube"
```

Comment:
@prietyc123 this should be CI_INFO="minishift"

Reply:
Defining it locally is also not helping much 🙁 . Looking more into it.

Reply:
@prietyc123  try using $CI_INFO instead of ${CI_INFO} in minikube-minishift-all-tests.sh and minikube-minishift-setup-env.sh